### PR TITLE
Release 3.0 -  tests ddp client and minimongo

### PR DESCRIPTION
**Pull request by denihs**
In this PR we:

- Fix tests in the ddp-client
image

- Fix tests in the minimongo
image

- keep Meteor.call and Meteor.appy sync

### CODE REVIEW

File: packages/autoupdate/autoupdate_server.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ Autoupdate.autoupdateVersionRefreshable = null;
 Autoupdate.autoupdateVersionCordova = null;
 Autoupdate.appId = __meteor_runtime_config__.appId = process.env.APP_ID;
 
-var syncQueue = new Meteor._SynchronousQueue();
+var syncQueue = new Meteor._AsynchronousQueue();
```
Comment (Grubba27) [inline packages/autoupdate/autoupdate_server.js:55]: Maybe we should name it asyncQueue or just queue

File: packages/ddp-client/test/livedata_connection_tests.js [hunk 2]
```diff
@@ -746,11 +772,11 @@ Tinytest.add('livedata stub - method call before connect', function(test) {
   });
 });
 
-Tinytest.addAsync('livedata stub - reconnect', async function(test, onComplete) {
+Tinytest.addAsync('livedata stub - reconnect111', async function(test, onComplete) {
```
Comment (Grubba27) [inline packages/ddp-client/test/livedata_connection_tests.js:775]: maybe typo ?

it could be just reconnect

**Review by Grubba27 — APPROVED** [review]
Looks good. 
Just nitpicks

File: packages/autoupdate/autoupdate_server.js (hunk 1 above)
Comment (zodern) [reply packages/autoupdate/autoupdate_server.js:55]: The queue only allows a single task to run at a time, which is why it was called a synchronous queue. The name wasn't referring to if the tasks themselves were sync or async; it always supported async tasks.

Comment (Grubba27) [reply packages/autoupdate/autoupdate_server.js:55]: ohhhhhh got it. Thanks for the clarification

File: packages/ddp-client/test/livedata_connection_tests.js (hunk 2 above)
Comment (denihs) [reply packages/ddp-client/test/livedata_connection_tests.js:775]: yes, typo

**Pull request merged**